src/test/java/org/mariadb/jdbc/BaseTest.java

```diff
@@ -27,7 +27,7 @@
  */
 @Ignore
 public class BaseTest {
-    protected static final String mDefUrl = "jdbc:mysql://localhost:3306/testj?user=root";
+    protected static final String mDefUrl = "jdbc:mysql://localhost:3306/testj?user=root&password=root";
```

Comment:
oops, sorry for this commit I will correct it immédiate !!

On Fri, Mar 4, 2016 at 7:21 PM, diego dupin <notifications@github.com>
wrote:

> In src/test/java/org/mariadb/jdbc/BaseTest.java
> <https://github.com/MariaDB/mariadb-connector-j/pull/72#discussion_r55067125>
> :
>
> > @@ -27,7 +27,7 @@
> >   */
> >  @Ignore
> >  public class BaseTest {
> > -    protected static final String mDefUrl = "jdbc:mysql://localhost:3306/testj?user=root";
> > +    protected static final String mDefUrl = "jdbc:mysql://localhost:3306/testj?user=root&password=root";
>
> please remove this, travis test consider having no password.
> for your tests, you can add
> -DdbURL=jdbc:mysql://localhost:3306/testj?user=root&password=root
> that will replace this default string
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MariaDB/mariadb-connector-j/pull/72/files#r55067125>.
>